engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -623,3 +623,9 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervi
 
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.vm_instance', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for vm" ');
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for volumes" ');
+
+-- Add console_endpoint_creator_address column to cloud.console_session table
+CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.console_session', 'console_endpoint_creator_address', 'VARCHAR(45)');
+
+-- Add client_address column to cloud.console_session table
+CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.console_session', 'client_address', 'VARCHAR(45)');
```

Comment:
these changes need to be moved to schema-42000to42010.sql or schema-42010to42100.sql

Reply:
@weizhouapache @JoaoJandre, I've just added the upgrade path from `4.20.1.0` to `4.21.0.0` (see 77530fc63e1a12e129e9dcd901eaef5f23772805). Could you guys take a look, please?